src/objective-c/BoringSSL-GRPC.podspec

```diff
@@ -212,7 +213,11 @@ Pod::Spec.new do |s|
     # limit on the 'prepare_command' field length. The encoded header is generated from
     # /src/boringssl/boringssl_prefix_symbols.h. Here we decode the content and inject the header to
     # the correct location in BoringSSL.
-    base64 -D <<EOF | gunzip > src/include/openssl/boringssl_prefix_symbols.h
+    case "$(uname)" in
+      Darwin) opts="" ;;
+           *) opts="--ignore-garbage" ;;
```

Comment:
I think overall this PR looks good, but what was the "garbage" that needed to be ignored when on linux?  Was it the newlines/spaces?

Reply:
Thanks Jan. Yes exactly, for linux variation (https://linux.die.net/man/1/base64) by default it does not handle non-alphabet bytes in the encode string, in this case the newline inside the podspec.  On Darwin base64 this is automatically handled by default (https://www.unix.com/man-page/osx/1/base64/)